src/convertToObject.js

```diff
@@ -7,6 +7,29 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const regex = /\n+|\t+/g;
+
+  sourceString
+    .replaceAll(regex, ' ')
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item !== '')
+    .map((item) => {
+      return item.split(':').map((x) => {
+        return x
+          .split(',')
+          .map((y, index) => {
+            return index === 0 ? y.trim() : `          ${y.trim()}`;
+          })
+          .join(',\n');
+      });
+    })
+    .forEach((element) => {
+      result[`${element[0]}`] = element[1];
```

Comment:
[CODE KNOWLEDGE] - No need to use template literals here for `element[0]`. You can directly use `result[element[0]] = element[1];`.